.github/Templates/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+### Description
```

Comment:
Since we don't have a level-two header, we should not jump straight to level three just to change the font size. Go ahead and use the level two header (`##`) for these level three headings. Basically, headings should describe the document structure, like would be seen in a table of contents, rather than to control its appearance.